src/types/styled-components.d.ts

```diff
@@ -2,5 +2,6 @@ import 'styled-components';
 import { Theme } from '../shared/theme.types';
 
 declare module 'styled-components' {
-  export type DefaultTheme = Theme;
```

Comment:
@JaimeFLandbot typescript does not like that way 🤔

Reply:
Styled-components should be done something under the hood. Nevermind, it's not as important.